docs/content/configuration/broker.md

```diff
@@ -86,6 +86,7 @@ See [groupBy server configuration](../querying/groupbyquery.html#server-configur
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.query.segmentMetadata.defaultHistory`|When no interval is specified in the query, use a default interval of defaultHistory before the end time of the most recent segment, specified in ISO8601 format. This property also controls the duration of the default interval used by GET /druid/v2/datasources/{dataSourceName} interactions for retrieving datasource dimensions/metrics.|P1W|
+|`druid.query.segmentMetadata.defaultAnalysisTypes`|This can be used to set the Default Analysis Types for all segment metadata queries, this can be overridden when making the query|[CARDINALITY, INTERVAL, MINMAX]|
```

Comment:
The default value here should be updated to `["cardinality", "interval", "minmax"]`. Also please point to this config in docs of Segment Metadata Queries, line 35.